### PR TITLE
Add mender params

### LINKED ISSUE #129: Add parameters to override values in mender.conf
At least we need these:
MENDER_UPDATE_POLL_INTERVAL_SECONDS ?= "1800"
MENDER_INVENTORY_POLL_INTERVAL_SECONDS ?= "1800"
MENDER_RETRY_POLL_INTERVAL_SECONDS ?= "300"

See https://github.com/tolstoyevsky/pieman/pull/94/files#r234751735

### PR DESCRIPTION
**Pull request by mdanil007**
close #129

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,7 +208,7 @@ The device the target image is created for is specified via the `DEVICE` environ
 
 ##### OS="raspbian-stretch-armhf"
 
-Allows specifying the operating system to be used as a base for the target image. You can find all the possible values for the parameter in the table above (see the "Short name" column). 
+Allows specifying the operating system to be used as a base for the target image. You can find all the possible values for the parameter in the table above (see the "Short name" column).
```
Comment (eugulixes) [inline README.md]: Let's avoid the changes.

File: README.md [hunk 2]
```diff
@@ -393,7 +393,7 @@ Note, that the parameter must follow the format "uid:gid" where `uid` and `gid`
 
 ##### PYTHON="$(which python3)"
 
-Allows specifying the Python 3 interpreter to be used when calling the Pieman-specific utilities. 
+Allows specifying the Python 3 interpreter to be used when calling the Pieman-specific utilities.
```
Comment (eugulixes) [inline README.md]: And we should avoid these changes as well.

**Comment by denismosolov** [comment]
What if we remove `_SECONDS`?
Ex.
`MENDER_INVENTORY_POLL_INTERVAL_SECONDS` -> `MENDER_INVENTORY_POLL_INTERVAL`
`MENDER_RETRY_POLL_INTERVAL_SECONDS` -> `MENDER_RETRY_POLL_INTERVAL`
`MENDER_UPDATE_POLL_INTERVAL_SECONDS` -> `MENDER_UPDATE_POLL_INTERVAL`
@mdanil007 what do you think?

File: README.md [hunk 3]
```diff
@@ -423,6 +423,14 @@ Specifies the size in megabytes of the data partition.
 
 The parameter is used only when `ENABLE_MENDER` is set to `true`.
 
+##### MENDER_INVENTORY_POLL_INTERVAL=86400
+
+Sets the frequency for periodically sending inventory data. Inventory data is always sent after each boot of the device, and after a new update has been correctly applied and committed by the device in addition to this periodic interval. Default value: 86400 seconds (one day).
+
+##### MENDER_RETRY_POLL_INTERVAL=300
+
+An integer that sets the number of seconds to wait between each attempt to download an update file. Note that the client may attempt more often initially to enable rapid upgrades, but will gradually fall back to this value if the server is busy. See also the section about polling intervals.
```
Comment (eugulixes) [inline README.md]: `See also [MENDER_INVENTORY_POLL_INTERVAL](#mender_inventory_poll_interval86400).`

File: README.md [hunk 4]
```diff
@@ -423,6 +423,14 @@ Specifies the size in megabytes of the data partition.
 
 The parameter is used only when `ENABLE_MENDER` is set to `true`.
 
+##### MENDER_INVENTORY_POLL_INTERVAL=86400
+
+Sets the frequency for periodically sending inventory data. Inventory data is always sent after each boot of the device, and after a new update has been correctly applied and committed by the device in addition to this periodic interval. Default value: 86400 seconds (one day).
```
Comment (eugulixes) [inline README.md]: `Sets the frequency (in seconds)...`

File: README.md [hunk 5]
```diff
@@ -433,6 +441,11 @@ The parameter is used only when `ENABLE_MENDER` is set to `true`.
 
 Specifies the server for the client to connect to.
 
+##### MENDER_UPDATE_POLL_INTERVAL=1800
+
+Sets the frequency the client will send an update check request to the server. Default value: 1800 seconds (30 minutes).
+
+
```
Comment (eugulixes) [inline README.md]: Redundant a new line.

File: README.md [hunk 6]
```diff
@@ -433,6 +441,11 @@ The parameter is used only when `ENABLE_MENDER` is set to `true`.
 
 Specifies the server for the client to connect to.
 
+##### MENDER_UPDATE_POLL_INTERVAL=1800
+
+Sets the frequency the client will send an update check request to the server. Default value: 1800 seconds (30 minutes).
```
Comment (eugulixes) [inline README.md]: `Sets the frequency (in seconds)...`

**Pull request merged**